include/libcyphal/transport/can/transport.hpp

```diff
@@ -226,6 +237,11 @@ class TransportImpl final : public ICanTransport, private TransportDelegate  //
                               CANARD_NODE_ID_MAX + 1};
     }
 
+    void setTransientErrorHandler(TransientErrorHandler handler) override
```

Comment:
```suggestion
    void setTransientErrorHandler(const TransientErrorHandler& handler) override
```

Reply:
This was done deliberately like this. Notice that all I do is just store the handler, so in such case "by value and then move" is normally recommended approach.